src/main/resources/db/changelog/init/15062021_create_location_discount_table.yml

```diff
@@ -0,0 +1,27 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1234567890-8
```

Comment:
let's use meaningful ids for changelogs
like this one - "15062021_create_location_discount_table"